projects/ngx-prx-styleguide/src/lib/fancy-form/button.component.css

```diff
@@ -87,7 +97,24 @@ button.dropdown-toggle.open::after {
 }
 
 div.dropdown-menu {
+  /* position to :host */
   position: absolute;
+  right: 0;
+  z-index: 997;
+  box-shadow: 0 5px 15px -5px rgba(0,0,0,.5);
+}
+
```

Comment:
this is a new overlay so that when you click outside the button dropdown it closes the dropdown